dallinger/recruiters.py

```diff
@@ -1,9 +1,8 @@
 """Recruiters manage the flow of participants to the experiment."""
-
 import os
 
 from boto.mturk.connection import MTurkConnection
-from psiturk.models import Participant
+from dallinger.models import Participant
```

Comment:
I think you're right. @jessesnyder was there a motivation for this change other than moving away from Psiturk (which should be covered by story 142 and friends)?